mods/ra/maps/soviet-05/soviet05.lua

```diff
@@ -206,14 +206,16 @@ WorldLoaded = function()
 	Trigger.OnDamaged(mcvtransport, Expand)
 
 	Trigger.OnKilled(Radar, function()
```

Comment:
At this point the Radar belongs to the AI. The situation you describe is handled [here](https://github.com/OpenRA/OpenRA/pull/12863/files#diff-28ca1ed8051b1dd53a820645c0bfac2cR232).